nestedquoteremover-me/src/compose.js

```diff
@@ -77,14 +77,29 @@ function processHtmlBlockquotes(nodes, replyHeaderHandling, maxDepth, depth) {
     for (let node of nodes) {
         if (node.nodeType === Node.ELEMENT_NODE &&
             node.nodeName === "BLOCKQUOTE" &&
-            node.hasAttribute("type") && node.getAttribute("type") === "cite") {
+            (node.hasAttribute("type") && node.getAttribute("type") === "cite" || // regular quote
+            node.hasAttribute("class") && node.getAttribute("class") === "gmail_quote")) // gmail web interface quote
+        {
             if (depth > maxDepth) {
                 node.remove();
             } else {
                 processHtmlBlockquotes(node.childNodes, replyHeaderHandling, maxDepth, depth + 1);
             }
         }
 
+        else if (node.nodeType === Node.ELEMENT_NODE &&
+            node.nodeName === "DIV" &&
+            node.hasAttribute("class") && node.getAttribute("class") === "gmail_quote") // gmail web interface pre-quote div
+        {
+            processHtmlBlockquotes(node.childNodes, replyHeaderHandling, maxDepth, depth);
+        }
+
+        else if (node.nodeType === Node.ELEMENT_NODE &&
+            node.nodeName === "DIV") // generic div
```

Comment:
@ValdikSS : I'm not sure that this last else-if is a bit too generic.
Basically this would mean, that ANY node that is a DIV will get treated as a blockquote.
Which is not good.
I tend to remove this last else block, but keep the other ones.

Reply:
Check my email example, it won't work correctly without this code. If you can make it better, go ahead.